src/debug_commands.c

```diff
@@ -1478,6 +1478,7 @@ DEBUG_COMMAND(WorkerThreadsSwitch) {
     // Log that we're waiting for the workers to finish.
     RedisModule_Log(RSDummyContext, "notice", "Debug workers drain");
 
+    //TODO(Joan): Should we remove this debug (or should we add documentation about the danger?)
```

Comment:
I don't think we should remove it for debug, I would go with documenting the danger

Reply:
how should we go about documenting this danger?